### ISSUE TITLE
Establish criteria for validating transformed data against OMOP CDM expectations

### ISSUE THREAD

**Issue by tursynay** [cladteam/fhir2omop-pm#16]
Establish criteria for validating transformed data against OMOP CDM expectations

**Comment by tursynay** [comment]
Check with Stephanie, how is the pipeline coming along

**Comment by tannerzhang** [comment]
OMOP CDM conformance check, down through enclave functionalities. Suggest waiting on hold until we have real/sample patient data there? @stephanieshong

**Comment by tursynay** [comment]
This is one of the pipeline steps (1 of 10). One of the steps to check OMOP CDM conformance.
